Content.Server/StationEvents/Components/VentCrittersRuleComponent.cs

```diff
@@ -14,4 +15,15 @@ public sealed partial class VentCrittersRuleComponent : Component
     /// </summary>
     [DataField("specialEntries")]
     public List<EntitySpawnEntry> SpecialEntries = new();
+
+
+    // Moffstation - Start - New variables for single vent spawn
+    public EntityCoordinates? Location;
```

Comment:
Move "runtime data" like this location below "configuration data" like `SpawnChances` and `SpecialEntries`.